templates/articles/_comment.html

```diff
@@ -0,0 +1,34 @@
+{% if view.comment_deleted %}
+    <turbo-frame id="comment-{{ view.kwargs.pk }}">
+     <!-- Comment Deleted -->
+    </turbo-frame>
```

Comment:
Should this be a stream response? Like:

```
<turbo-stream action="remove" target="comment-{{ view.kwargs.pk }}">
</turbo-stream>
```

Reply:
Ohh, I totally forgot that was a thing without websockets. Let me give that a go.

Reply:
That works much better than my blank trick. I just had to set the header on the view. That can be swapped out when we have a definitive set of mixins, similar to: https://github.com/hotwire-django/django-turbo-response/blob/main/src/turbo_response/response.py